browser/resources/settings/brave_privacy_page/brave_tor_subpage.html

```diff
@@ -0,0 +1,65 @@
+<style include="settings-shared md-select">
+  img {
+    width: 100%;
+  }
+
+  #torBridgesControls {
+    padding: 0 var(--cr-section-padding);
+  }
+
+  #requestedBridges,
+  #providedBridges {
+    width: 90%;
+    min-height: 70px;
+    height: auto;
+    resize: vertical;
+  }
+
+  textarea {
+    white-space: pre;
+    overflow-wrap: normal;
+    overflow-x: scroll;
+  }
+</style>
+<div id="torBridgesControls">
+  <settings-toggle-button id="useBridges" class="hr" label="$i18n{torUseBridgesTitle}"
+    sub-label="$i18n{torUseBridgesDesc}" learn-more-url="https://tb-manual.torproject.org/en-US/bridges/"
+    checked="[[isUsingBridges_]]" on-change="onUseBridgesChange_">
+  </settings-toggle-button>
+  <settings-radio-group id="brigdesGroup" no-set-pref selectable-elements="settings-collapse-radio-button"
```

Comment:
Seems like we can use `<cr-radio-group>` here as settings-radio-group just wraps to add pref binding which isn't being used here?